src/entities/addLiquidity/weighted/addLiquidityWeighted.ts

```diff
@@ -143,7 +146,9 @@ export class AddLiquidityWeighted implements AddLiquidityBase {
                 };
             }
             default:
-                throw Error('Unsupported Add Liquidity Kind');
+                throw Error(
+                    'Unsupported Add Liquidity Kind, for Init use InitPool instead of AddLiquidity',
+                );
```

Comment:
For the default case, I'd go with a generic error message, without mentioning Init:
```suggestion
                throw Error(
                    'Unsupported Add Liquidity Kind: ${input.kind}',
                );
```